birdhouse/config/geoserver/entrypointwrapper

```diff
@@ -1,20 +1,6 @@
 #!/bin/sh -x
```

Comment:
The file seems to have been removed from the docker service volumes/entrypoint. 
Is it not needed anymore?

Reply:
Will remove once I can test we do not need.  Had too many problems synching prod data to test VM so still not able to test this yet.